libraries/RC_Channel/RC_Channel.h

```diff
@@ -64,6 +64,8 @@ class RC_Channel {
     int16_t    get_radio_in() const { return radio_in;}
     void       set_radio_in(int16_t val) {radio_in = val;}
 
+    int16_t    get_radio_in_underlying() const { return radio_in_underlying;}
```

Comment:
It's a funny name perhaps, "radio_in_underlying".. maybe we can come up with a name that is more easily understood?  Perhaps, "get_radio_in_last_good_value()"?

Reply:
There was only supposed to be a single call to this function in each vehicle; this wasn't the last good value, rather the one read from the receiver.

Sadly, this PR fell in a heap as relying on the vehicle code to tell you the throttle value is bad is only going to happen after you've just set body values for each of the inputs.